common/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -225,6 +227,28 @@ public synchronized void connect() throws IOException {
   }
 
   /**
+   * Create transport per the connection options Supported transport options are: NOSASL, SIMPLE,
+   * CUSTOM, KERBEROS
+   * @throws TTransportException
+   */
+  private TTransport createTransport() throws IOException {
```

Comment:
Can we put this method in a helper class, maybe `PlainSaslHelper` ? The implementation should be not in this `MasterClient` class.